src/v/transform/logging/tests/log_manager_test.cc

```diff
@@ -181,7 +181,7 @@ TEST_F(TransformLogManagerTest, EnqueueLogs) {
 TEST_F(TransformLogManagerTest, LogsDroppedAtShutdown) {
     constexpr size_t n = 10;
 
-    for (int i = 0; i < n; ++i) {
+    for (size_t i = 0; i < n; ++i) {
```

Comment:
implicit signed narrowing conversion in the unit tests is how you know I wrote this code ✅

Reply:
Haha this warning (and the implicit conversions) is such a pain TBH. I think we have warnings consistent in Bazel for library code and tests, where in CMake this warning is only on for library code (not tests)

Reply:
yeah. wish they were just compiler errors honestly 🦀